src/drive/web/modules/certifications/index.jsx

```diff
@@ -0,0 +1,7 @@
+import { useQuery } from 'cozy-client'
+
+export const MakeConditionWithQuery = ({ query }) => {
+  const queryResult = useQuery(query.definition, query.options)
+
+  return queryResult.fetchStatus === 'loaded' && queryResult.data.length > 0
+}
```

Comment:
This is surprising, this is using a hook but this is not a component.

Do you want to directly use the client.store here ?

client.getQueryFromState can be used to retrieve the data stored in cozy-client.